setup.py

```diff
@@ -39,7 +39,7 @@ def _dbt_duckdb_version():
     install_requires=[
         "dbt-common>=1,<2",
         "dbt-adapters>=1,<2",
-        "duckdb>=0.7.0,!=0.10.3",
+        "duckdb>=0.10.0,!=0.10.3",
```

Comment:
chiming in here from adapters, you are totally free to version as you see fit. We generally recommend that major version bumps reflect _major_ changes to the adapter and should therefore be reserved. Minor versions should reflect meaningful changes to behavior.
Beyond versioning we do want to encourage adapters to keep it easy for folks to upgrade and to not surprise people with changes. To do that we want to introduce breaking behavioral changes as opt-in with a warning that the default behavior will change in a future version. We are going to be picking up a project in Q3 to add [behavior flags](https://github.com/dbt-labs/dbt-adapters/issues/248) to formalize this process. This should provide a framework for adapters to manage the life cycle of these changes.

Reply:
thanks Colin, super-helpful context here

Reply:
Happy to bump to 1.0.0 in a separate PR, I want to make sure there's a commit that still has 0.10.2 included in case someone turns up with that requirement

Reply:
Happy to bump to 1.0.0 in a separate PR, I want to make sure there's a commit that still has 0.10.2 included in case someone turns up with that requirement

Reply:
PR open [here](https://github.com/duckdb/dbt-duckdb/pull/417)!